pkg/lib/database/filter_test.go

```diff
@@ -26,6 +26,15 @@ func assertFilter(t *testing.T, f Filter, obj *types.Struct, expected bool) {
 	assert.Equal(t, expected, anystoreFilter.Ok(val))
 }
 
+func assertFilterNoParseValidation(t *testing.T, f Filter, obj *types.Struct, expected bool) {
```

Comment:
change original function please, this parsing validation is not necessary

Reply:
Yeah, removed parse validation from filter assertion